qiskit-superstaq/qiskit_superstaq/serialization.py

```diff
@@ -15,11 +16,11 @@
 import qiskit_superstaq as qss
 
 T = TypeVar("T")
-RealArray = Union[int, float, List["RealArray"]]
+RealArray = TypeVar("RealArray")
```

Comment:
it's probably better to just delete this and change the annotation for `json_encoder` to `-> dict[str, object]` (we won't be able to use the correct alias again until we drop support for python 3.8)

Reply:
Sounds good, done